core/src/main/java/com/datastrato/gravitino/storage/relational/mysql/mapper/MetalakeMetaMapper.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+
+package com.datastrato.gravitino.storage.relational.mysql.mapper;
+
+import com.datastrato.gravitino.storage.relational.mysql.po.MetalakePO;
+import java.util.List;
+import org.apache.ibatis.annotations.Delete;
+import org.apache.ibatis.annotations.Insert;
+import org.apache.ibatis.annotations.Param;
+import org.apache.ibatis.annotations.Select;
+import org.apache.ibatis.annotations.Update;
+
+public interface MetalakeMetaMapper {
```

Comment:
Why do we use interface here? Could you add some comments for this class?

Reply:
This is a specification defined by MyBatis. It requires this interface class to identify the corresponding SQL for execution. We can write SQLs in an additional XML file, or write SQLs in the annotation which in this interface Mapper, see: https://mybatis.org/mybatis-3/getting-started.html. I'll add the comment for this.